TMDBApp/TMDBApp/General.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Geral.swift
+//  TMDBApp
+//
+//  Created by Richard Leh on 19/12/2016.
+//  Copyright © 2016 Richard Leh. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SVProgressHUD
+
+typealias CompletionResultError = (_ result: Any?, _ error: Error?) -> Void
+typealias DictionaryData = Dictionary<String, Any>
+
+func updatesOnMain(_ updatesToMake: @escaping () -> Void) {
+    DispatchQueue.main.async {
+        updatesToMake()
+    }
+}
+
+func genrerFormatedString(fromMovie movie:Movie) -> String{
+    if let mGenres = movie.genres {
+        
+        var genresString = ""
+        
+        let genresNames = mGenres.map { $0.name }
+        for (index, name) in genresNames.enumerated(){
```

Comment:
I could use `genresNames.joined(separator: ", ")` but I need to change the genresString to array with not optional String.